src/debugger/debugAdapterFactory.ts

```diff
@@ -91,14 +91,19 @@ function registerLLDBDebugAdapter(
 export class LLDBDebugConfigurationProvider implements vscode.DebugConfigurationProvider {
     constructor(
         private platform: NodeJS.Platform,
-        private toolchain: SwiftToolchain,
+        private workspaceContext: WorkspaceContext,
         private outputChannel: SwiftOutputChannel
     ) {}
 
     async resolveDebugConfigurationWithSubstitutedVariables(
         _folder: vscode.WorkspaceFolder | undefined,
```

Comment:
Should probably remove the underscore here since it was only there to silence the unused argument warning.
```suggestion
        folder: vscode.WorkspaceFolder | undefined,
```